src/Logic/SubtitleFormats/SubtitleFormat.cs

```diff
@@ -363,35 +369,27 @@ public virtual bool HasStyleSupport
 
         public bool BatchMode { get; set; }
 
-        public static string ToUtf8XmlString(XmlDocument xml, bool omitXmlDeclaration)
+        public static string ToUtf8XmlString(XmlDocument xml, bool omitXmlDeclaration = false)

```

Comment:
The behaviour of `ToUtf8XmlString()` should be exactly the same as before. I will investigate this later.
For now, I shall remove this commit from the PR.